docs/contribute/redirects.md

```diff
@@ -0,0 +1,101 @@
+---
+navigation_title: "Redirects"
+---
+# Adding redirects to link resolving
+
+If you [move files around](move.md) or simply need to delete a few pages you might end up in a chicken-and-egg situation. The files you move or delete might still be linked to by other [documentation sets](../configure/content-set/index.md)
+
+Redirects allow you to force other documentation sets to resolve old links to their new location.
+This allows you to publish your changes and work to update the other documentation sets.
```

Comment:
assume this will also just add normal redirect functionality (sorry for the worst sentence ever 🤷)

```suggestion
This allows you to publish your changes and work to update the other documentation sets.

Redirects also allow any external links to our documentation impacted by a reorganization to resolve instead of returning a 404.
```

Reply:
> Redirects also allow any external links to our documentation impacted by a reorganization to resolve instead of returning a 404.

This is not true yet, this currently only affects our link rewriting. We don't generate redirect mappings (yet).

Reply:
we might want to say this in a note - that the redirects won't resolve for now but will in future(?)

Reply:
Agreed that we should note this. I'll add in a follow-up PR so we can get this merged.